Boards/raspberrypi_pico.board.json

```diff
@@ -0,0 +1,199 @@
+{
+  "$schema": "./mfboard.schema.json",
+  "UsbDriveSettings": {
```

Comment:
The presence of `UsbDriveSettings` in the board config file will trigger that flash method instead of using AvrDude. Way back when I added board configs I planned for multiple flashing methods so this fit right in.